source/fuzz/fuzzer_pass_donate_modules.cpp

```diff
@@ -1044,7 +1046,87 @@ void FuzzerPassDonateModules::AddLivesafeFunction(
       loop_limiter.set_increment_id(GetFuzzerContext()->GetFreshId());
       loop_limiter.set_compare_id(GetFuzzerContext()->GetFreshId());
       loop_limiter.set_logical_op_id(GetFuzzerContext()->GetFreshId());
-      loop_limiters.emplace_back(loop_limiter);
+
```

Comment:
Consider changing the above `if (block.IsLoopHeader()) {` to 

```
if (!block.IsLoopHeader()) {
  continue;
}
```

to reduce indentation.

Reply:
I've refactored the code into a separate function in d4d7640, thus decreasing the indentation. Hence, I left this block as it is.